code/datums/spells/ethereal_jaunt.dm

```diff
@@ -95,6 +95,7 @@
 	var/canmove = 1
 	density = 0
 	anchored = 1
+	invisibility = INVISIBILITY_OBSERVER

```

Comment:
https://github.com/tgstation/-tg-station/commit/8a26ff252754a1691eb9d8332fde20121e4aa9eb it was here..but i did it wrong..I will just revert it to server values....*slaps self*